### DISCUSSION TITLE
How to properly configure Subscriptions file and what command to acess subscriptions? (linux Beginner)

### DISCUSSION THREAD

**Discussion by ttx99** [pystardust/ytfzf#622] [Q&A] [answered]
I have read the documentation and Im still confused. my subscription file is in: /home/mac99/.config/ytfzf/subscriptions.sh.
I dont know what file is it a txt file or sh?
what is the command in ytfzf to display the videos of my subscriptions?

**Comment by Euro20179** [comment]
> `/home/mac99/.config/ytfzf/subscriptions.sh`

It's called `subscriptions` not `subscriptions.sh`

> what is the command in ytfzf to display the videos of my subscriptions?
`ytfzf -cSI`

**Comment by ttx99** [comment]
it shows this

![Screenshot_2023-01-10_14-18-54](https://user-images.githubusercontent.com/95531944/211476139-eee3a9bd-c37f-4eab-8a94-bcaee4492f89.png)

**Reply by Euro20179** [reply]
just put `@DotaCinema` instead of a link

**Comment by ttx99** [comment, marked as answer]
it's works now somehow, but some channels are not scraped? Is this just the limitation of the scraper?

**Reply by Euro20179** [reply]
You're probably putting an invalid link for the ones that aren't being scraped. Try going to the youtube page for that channel and seeing what the link is.

If its in the form of `https://www.youtube.com/c/<name>`, `httos://www.youtube.com/user/<name>`, or `https://www.youtube.com/channel/<id>` then you should put the full link.


I think the reason the full link doesnt work for `@channel`, is simply that I never made it work,  a fix for that if im right will be in the next update.

edit:
For some channels, getting the real id of the channel the way im doing doesn't work, this only happens for channels with an `@` link.

edit 2:
Fixed the bug, if you want to use the development branch.

**Comment by ttx99** [comment]
channel with @link don't work all the time (for very popular channels it works). So I used this website https://commentpicker.com/youtube-channel-id.php to get the exact channel URL and now everything works.
